baremetal_network_provisioning/ml2/hp_network_provisioning_driver.py

```diff
@@ -34,7 +34,7 @@
     cfg.StrOpt('base_url',
                help=_("Base HTTP URL of  SDN controller")),
     cfg.StrOpt('auth_token',
-               default='AuroraSdnToken37',
+               default='password',
```

Comment:
Hi Vivek,
Though we are reading the auth_token from the ini file , this change is to accommodate the default value in case auth_token is not configured in ini file by user.
If user is not configured for auth_token then ‘password ’ will be used in this case.


Thanks
Selvakumar.

From: hp-networking [mailto:notifications@github.com]
Sent: Friday, November 06, 2015 6:01 PM
To: hp-networking/baremetal-network-provisioning
Subject: Re: [baremetal-network-provisioning] Remove Default Hardcoded auth token (#24)


In baremetal_network_provisioning/ml2/hp_network_provisioning_driver.py<https://github.com/hp-networking/baremetal-network-provisioning/pull/24#discussion_r44131734>:

> @@ -34,7 +34,7 @@

>      cfg.StrOpt('base_url',

>                 help=_("Base HTTP URL of  SDN controller")),

>      cfg.StrOpt('auth_token',

> -               default='AuroraSdnToken37',

> +               default='password',

Better to have the password read from an input file.

How do we expect the customer to change this in code?

—
Reply to this email directly or view it on GitHub<https://github.com/hp-networking/baremetal-network-provisioning/pull/24/files#r44131734>.